repo/blob/s3/s3_storage_test.go

```diff
@@ -231,6 +245,32 @@ func testStorage(t *testutil.RetriableT, options *Options) {
 	}
 }
 
+func TestCustomTransportNoSSLVerify(t *testing.T) {
+	testURL(expiredBadSSL, t)
+	testURL(selfSignedBadSSL, t)
+	testURL(untrustedRootBadSSL, t)
+	testURL(wrongHostBadSSL, t)
+	testURL(incompleteChainBadSSL, t)
+}
+
+func getURL(url string, insecureSkipVerify bool) error {
+	client := &http.Client{Transport: getCustomTransport(insecureSkipVerify)}
+	_, err := client.Get(url)
+	return err
+}
+
+func testURL(url string, t *testing.T) {
+	err := getURL(url, true)
+	if err != nil {
+		t.Fatalf("could not get url:%s, error:%v", url, err)
```

Comment:
@ntolia - Are you referring to the error strings here and at L270 in ur recent comment ? 

At L 265, since it is the scenario where we disable tls verification, the error string should say it is a credentials issue? (But there are no credentials required for getting the Bad SSL URLs)
And at L270 since we don't disable tls verification, we want the error to be "TLS issue detected" ?

Reply:
OR you are referring to the code in testStorage():
https://github.com/kastenhq/kopia/blob/becf08af8a41f87101c269a19319acf04f55d77f/repo/blob/s3/s3_storage_test.go#L222 

We could  add the options.DoNotVerifyTLS in the error message  to help debug failures.

Reply:
@onkarbhat Actually, wasn't referring to any of those particular lines but they might work if it is easy to evaluate. If this is too hard to test, we can move on.